pyoptsparse/pyOpt_history.py

```diff
@@ -670,8 +673,26 @@ def getValues(self, names=None, callCounters=None, major=True, scale=False, stac
             if data[name].ndim == 1:
                 data[name] = np.expand_dims(data[name], 1)
 
+        # Raise warning for IPOPT's duplicated history
+        if self.db["metadata"]["optimizer"] == "IPOPT" and "iter" not in self.db["0"].keys():
```

Comment:
Should suggest a fix, telling the user to either manually fix or re-run the optimization with an updated version of pyOptSparse or something.